x/wasm/types/proposal_test.go

```diff
@@ -804,40 +804,6 @@ func TestUnmarshalContentFromJson(t *testing.T) {
 	}
 }
 
-func TestProposalJsonSignBytes(t *testing.T) {
```

Comment:
I think I deleted the test because it was duplicated in cherry-pick.

https://github.com/line/lbm-sdk/pull/570/files/aa6e12abed335fcfda88e52238e8fe394e8f6ec8..95f7d374ef53b288bb9736443183e9c80fdd00da#diff-f93e1bfff58369727113a783e468ed5b5bc5847006a9291d50863880590e4093L841